packages/twenty-server/src/engine/core-modules/billing/services/billing-webhook-customer.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+
+import Stripe from 'stripe';
+import { Repository } from 'typeorm';
+
+import { BillingCustomer } from 'src/engine/core-modules/billing/entities/billing-customer.entity';
+@Injectable()
+export class BillingWebhookCustomerService {
+  protected readonly logger = new Logger(BillingWebhookCustomerService.name);
+  constructor(
+    @InjectRepository(BillingCustomer, 'core')
+    private readonly billingCustomerRepository: Repository<BillingCustomer>,
+  ) {}
+
+  async processStripeCustomerEvent(
```

Comment:
this function should not return a workspaceId (you need it because you are putting some other logic depending in the controller). If it does we should make the naming clear